ui/tests/acceptance/storage-list-test.js

```diff
@@ -191,43 +179,33 @@ module('Acceptance | volumes list', function (hooks) {
       namespaceId: server.db.namespaces[1].id,
     });
 
-    await VolumesList.visit();
-    assert.equal(VolumesList.volumes.length, 2);
+    await StorageList.visit();
+    assert.equal(StorageList.csiVolumes.length, 2);
 
     const firstNamespace = server.db.namespaces[0];
-    await VolumesList.visit({ namespace: firstNamespace.id });
-    assert.equal(VolumesList.volumes.length, 1);
-    assert.equal(VolumesList.volumes.objectAt(0).name, volume1.id);
+    await StorageList.visit({ namespace: firstNamespace.id });
+    assert.equal(StorageList.csiVolumes.length, 1);
+    assert.equal(StorageList.csiVolumes.objectAt(0).name, volume1.id);
 
     const secondNamespace = server.db.namespaces[1];
-    await VolumesList.visit({ namespace: secondNamespace.id });
+    await StorageList.visit({ namespace: secondNamespace.id });
 
-    assert.equal(VolumesList.volumes.length, 1);
-    assert.equal(VolumesList.volumes.objectAt(0).name, volume2.id);
+    assert.equal(StorageList.csiVolumes.length, 1);
+    assert.equal(StorageList.csiVolumes.objectAt(0).name, volume2.id);
   });
 
   test('when accessing volumes is forbidden, a message is shown with a link to the tokens page', async function (assert) {
     server.pretender.get('/v1/volumes', () => [403, {}, null]);
 
-    await VolumesList.visit();
-    assert.equal(VolumesList.error.title, 'Not Authorized');
+    await StorageList.visit();
+    assert.equal(StorageList.error.title, 'Not Authorized');
 
-    await VolumesList.error.seekHelp();
+    await StorageList.error.seekHelp();
     assert.equal(currentURL(), '/settings/tokens');
   });
 
-  pageSizeSelect({
-    resourceName: 'volume',
-    pageObject: VolumesList,
-    pageObjectList: VolumesList.volumes,
-    async setup() {
-      server.createList('csi-volume', VolumesList.pageSize);
-      await VolumesList.visit();
-    },
-  });
```

Comment:
Feel comfortable deleting the pageSizeSelect test module from this particular test. It's tested in several other contexts and it's very stable (it's the thing that checks that our localStorage.nomadPageSize affects the tables you see on your page)